Add WARP templates (#2)
populationgenomics/cromwell-configs: README.md

Make sure to keep `&` in the end to run the process in the background, otherwise you might accidentally interrupt the execution. To bring the process to the foreground, so you could interrupt it, run `fg`.

## WARP inputs

> The `warp-inputs/` folder contains templates that can be modified to use with [WGS or WES germline variant calling WARP workflows](https://github.com/populationgenomics/warp/blob/master/pipelines/broad/dna_seq/germline/single_sample/). You'll have to replace the input parameters on top (`<sample-name>` and `<bam-location>` for the workflows that start from BAM, or the parametrs in the `sample_and_fastqs` section for the workflows that start from Fastqs); all the reference-data inputs are pre-filled to point to the Broad public genomcis buckets.

Reviewer: ```suggestion
The `warp-inputs/` folder contains templates that can be modified to use with [WGS or WES germline variant calling WARP workflows](https://github.com/populationgenomics/warp/blob/master/pipelines/broad/dna_seq/germline/single_sample/). You'll have to replace the input parameters at the top (`<sample-name>` and `<bam-location>` for the workflows that start from BAM, or the parameters in the `sample_and_fastqs` section for the workflows that start from FASTQs); all the reference-data inputs are pre-filled to point to the Broad public genomics buckets.
```
